pkcs11/signer_test.go

```diff
@@ -278,16 +291,21 @@ func TestSignX509Cert(t *testing.T) {
 	}
 	cp.AddCert(caCert)
 
+	timeoutCtx, cancel := context.WithTimeout(context.Background(), 10*time.Microsecond)
+	defer cancel()
+
 	testcases := map[string]struct {
+		ctx         context.Context
 		cert        *x509.Certificate
 		identifier  string
 		isBadSigner bool
 		expectError bool
 	}{
-		"cert-rsa-good-signer": {certRSA, defaultIdentifier, false, false},
-		"cert-ec-good-signer":  {certEC, defaultIdentifier, false, false},
-		"cert-bad-identifier":  {certRSA, badIdentifier, false, true},
-		"cert-bad-signer":      {certRSA, defaultIdentifier, true, true},
+		"cert-rsa-good-signer": {context.Background(), certRSA, defaultIdentifier, false, false},
```

Comment:
nit: can we initialize `ctx` above and use that variable in all test cases?

Reply:
Yes. Somehow I missed this test case(code was folded in IDE) when changing the others in this file to use ctx.